Truck.js

```diff
@@ -1,6 +1,31 @@
 var Car = require('./Car.js');
 
 //Implement your Truck here
-function Truck() {};
+function Truck(make, model, year, color, capacity) {
+	Car.call(this, make, model, year, color, 3);
```

Comment:
I think you're missing seats as one of your parameters, so I would remove the ```3``` from ```Car.call``` here and then assign seats using ```this.seats = 3```